Service Description changes for Pro (#12097)
canonical/ubuntu.com: templates/legal/ubuntu-pro-description/index.md

            9. Support must be purchased for all Nodes in the supported Kubernetes cluster
            10. Supported versions of Kubernetes include:
                4. Weekday or 24/7 Support for N-2 (the latest and previous two) releases in the stable release channel
                5. ESM security patching for N-4 (the latest and previous four) releases in the stable release channel
            11. For any deployment of Charmed Kubernetes carried out by Canonical while under contract for a deployment, which results in the customisation of any Charms, the customisation will be supported for 90 days after completion of the deployment
        7. <span id="uprosd-kubernetes-support">Limited Kubernetes Support</span>
            12. Unless the Full-Stack Support requirements set out above are met, support is limited to Bug-fix Support for:
                6. The software packages and Charms necessary for running Charmed Kubernetes
                7. Kubernetes clusters deployed using kubeadm utilising the software packages available from apt.kubernetes.io

> 3. <span id="uprosd-infra-only">Support &mdash; included in Infra and Full support, but not included in Apps support</span>:

Reviewer: ```suggestion
3. <span id="uprosd-infra-only">Support &mdash; included in Infra and Full Stack Support, but not included in Apps Support</span>:
```

        22. <span id="uprosd-maas-support-scope">MAAS support scope</span>:
            22. Support for the ability to boot machines using operating system images provided by Canonical
            23. Support for the tooling required to convert certified operating system images not provided by Canonical into MAAS images
        23. <span id="uprosd-out-of-scope">Out of scope</span>. MAAS support does not provide:
            24. Support for workloads, packages and service components other than those required to run a MAAS deployment
            25. Support for the servers that the MAAS service runs on
            26. Support for the Nodes deployed using MAAS but not covered under Ubuntu Pro
            27. Support for design and implementation details of a MAAS deployment
            28. Access to Landscape and Canonical Livepatch Service for machines deployed with MAAS
    12. <span id="uprosd-ubuntu-assurance-program">Ubuntu Assurance Program</span>
        24. Ubuntu Pro + support customers are entitled to the [Ubuntu Assurance Programme](http://www.ubuntu.com/legal/ubuntu-advantage/assurance). Canonical may update the Assurance Programme and its terms periodically

> 4. <span id="uprosd-apps-only">Support &mdash; included in Apps support and Full support, but not included in Infra support</span>

Reviewer: ```suggestion
4. <span id="uprosd-apps-only">Support &mdash; included in Apps Support and Full Stack support, but not included in Infra support</span>
```

<div class="p-top"><a href="#" class="p-top__link">Back to top</a></div>

<h2 id="uprosd-add-ons">Add Ons</h2>

<h3 id="uprosd-managed-services">Managed Services</h3>

> 1. Managed Services are an add-on to Ubuntu Pro + 24/7 support (for Infra support, Apps support and Full support). When added on, Canonical will manage the Environment as described below. Guests Instance running in the Environment receive the Ubuntu Advantage subscriptions and support

Reviewer: ```suggestion
1. Managed Services are an add-on to Ubuntu Pro + Infra Support (24/7), or Ubuntu Pro + Apps support (24/7) or Ubuntu Pro +  Full Stack support (24/7). When added on, Canonical will manage the Environment as described below. Guests Instance running in the Environment receive the Ubuntu Advantage subscriptions and support
```
